cordova/package-lock.json

```diff
@@ -0,0 +1,3715 @@
+{
```

Comment:
The Cordova CLI uses npm and not yarn. I will try to find a way to use yarn instead of npm but I am not sure it will be possible.

Reply:
I'm not an expert in `yarn` or `npm`, so maybe I'm saying a stupid thing but at the end, we use `yarn` like a wrapper of `npm` and it generates the `package-lock.json` in the same way. 
We only don't need to upload it because the information is at the `yarn.lock` file, that is what we upload. 
Have you tested to execute a `yarn install` to see if it generates the `package-lock.json` file?

Reply:
Sorry, reading your message again now I understand the problem. Is Cordova directly who uses NPM. Sorry to not understand you the first time.